multimodal/setup.py

```diff
@@ -32,24 +32,25 @@
     "seqeval<=1.2.2",
     "evaluate<=0.3.0",
     "accelerate>=0.9,<0.14",
+    "tensorboard<2.12.0",
     "timm<0.7.0",
-    "torch>=1.9,<1.13",
-    "torchvision<0.14.0",
-    "torchtext<0.14.0",
+    "torch>=1.9,<1.14",
```

Comment:
security concern: `1.12.x` have critical RCE CVEs open against it. pytorch is not planning to fix it in `1.12.x`, I suggest capping it from the bottom to `>=1.13.1`?
- https://snyk.io/vuln/SNYK-PYTHON-TORCH-3149871
- https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-45907.html
- github issue thread: https://github.com/pytorch/pytorch/issues/88868